### PR TITLE
Added description for Holding the invoice in purchase invoice

### PR DESCRIPTION
**Pull request by ShraddhaRanjane**
Holding the Invoice:-
I have added Description regarding Block and Unblock invoice in Purchase Invoice doc

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/accounts/purchase-invoice.md [hunk 1]
```diff
@@ -71,6 +71,12 @@ Sometimes you may need to hold an invoice from being submitted.
 
 ![PI Hold](/docs/assets/img/accounts/pi-hold.png)
 
+If you want to change the 'Release Date' after the Purchase Invoice is submitted, click on the **Hold Invoice** button on the top right.
+
+If you want to hold a submitted purchase invoice, click on **Create** then the **Block Invoice** option. If you want to unblock the invoice again then use 'Unblock Invoice' option.
+
+![Block PI](/docs/assets/img/accounts/pi_block.png)
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/purchase-invoice.md]: @ShraddhaRanjane, please highlight the Block Invoice option in the screenshot with a sharp border RED rectangle.
take care that the rectangle does not leave any empty spaces around it

Comment (ShraddhaRanjane) [reply erpnext_com/www/docs/user/manual/en/accounts/purchase-invoice.md]: Okay. I will do changes .

**Comment by ShraddhaRanjane** [comment]
@prasadarr I have updated screen-shot as per your suggestion.please check.

**Comment by prasadarr** [comment]
> @prasadarr I have updated screen-shot as per your suggestion.please check.

the screenshot quality needs work. It is cropped unevenly, has too much padding on the right and too less on the left, the padding needs to be even.

The red rectangle highlight should be like this, refer: https://erpnext.com/docs/user/manual/en/accounts/journal-entry#22-reverse-journal-entry

**Comment by ShraddhaRanjane** [comment]
@prasadarr  Updated image quality as per your suggestion.

**Comment by ShraddhaRanjane** [comment]
@prasadarr  Any update on this ?

**Pull request merged**